ols/app/models/config.py

```diff
@@ -301,6 +302,57 @@ def validate_yaml(self) -> None:
         """Validate memory cache config."""
 
 
+class Filter(BaseModel):
+    """Filter configuration."""
+
+    name: Optional[str] = None
+    regular_expression: Optional[str] = None
+    replace_with_string: Optional[str] = None
+
+    def __init__(self, data: Optional[dict] = None) -> None:
+        """Initialize configuration and perform basic validation."""
+        super().__init__()
+        if data is None:
+            return
+        try:
+            self.name = data.get("name", None)
```

Comment:
None is default value, not have to be specified (but it's used elsewhere anyway)